src/Tests/Microsoft.Diagnostics.Monitoring.Tool.UnitTests/Options/CollectionRuleOptionsExtensions.cs

```diff
@@ -143,6 +143,19 @@ public static CollectionRuleOptions AddCollectLiveMetricsAction(this CollectionR
                 });
         }
 
+        public static CollectionRuleOptions AddCollectStacksAction(this CollectionRuleOptions options, string egress = null, CallStackFormat? format = null)
```

Comment:
```suggestion
        public static CollectionRuleOptions AddCollectStacksAction(this CollectionRuleOptions options, string egress, CallStackFormat? format = null)
```

Egress should not be optional